mplane.docker

```diff
@@ -1,13 +1,13 @@
 # Make the mplane image from the monroe_base
 
-FROM monroe/base
+FROM monroe/base:cli
```

Comment:
The default for projects are to not use the :tags. Today :cli and monroe/base are the same but that may not be so in later revisions of monroe/base.